style.css

```diff
@@ -1,9 +1,10 @@
-font-family: 'DM Sans', sans-serif;
-font-family: 'Mulish', sans-serif;
+/*font-family: 'DM Sans', sans-serif;
```

Comment:
try write code with this guide https://codeguide.academy/html-css.html#css

Reply:
ok